SolarTherm/Models/Fluid/Valves/PBS_TeeJunction.mo

```diff
@@ -0,0 +1,40 @@
+within SolarTherm.Models.Fluid.Valves;
+model PBS_TeeJunction
```

Comment:
please explain why packed bed storage requires its own TeeJunction model -- via comments in the code.